@ecomplus/storefront-components/src/js/BuyTogether.js

```diff
@@ -165,12 +190,24 @@ export default {
         }
       }).finally(() => {
         this.hasLoadedIds = true
+        if (this.relatedProducts) {
+          this.productQnts = this.relatedProducts || []
+        }
         this.$nextTick(() => {
           if (!this.productIds.length) {
             this.hasLoadedItems = true
           }
```

Comment:
Só aqui você deveria testar produtos relacionados, primeiro configurados no corpo do produto, se ainda não houver tentaria a request em `/related.json`

Reply:
Ou seja, só depois de ver que não há produtos para "compre junto"

Reply:
E aí, se houver relacionados você seta `this.productQnts = ...`

Reply:
Eu já tinha pensado uma solução assim, mas como não funcionou, resolvi mudar pra ver se funcionava de outro jeito.